packages/website/src/features/Packages/interact/ContractsList.tsx

```diff
@@ -109,19 +139,55 @@ export const ContractsList: FC<ContractsListProps> = ({
                     onClick={async () => {
                       setIsPopoverOpen(false);
                       await router.push(
-                        `/packages/${name}/${tag}/${variant}/interact/${option.moduleName}/${option.contractName}/${option.contractAddress}`
+                        buildInteractPath(
+                          name,
+                          tag,
+                          variant,
+                          option.moduleName,
+                          option.contractName,
+                          option.contractAddress
+                        )
                       );
                     }}
-                    data-testid={`${option.moduleName}.${option.contractName}-button`}
+                    data-testid={`${option.contractName}-button`}
                   >
-                    <span className="text-sm">
-                      {`${option.moduleName}.${option.contractName}`}
+                    <span className="text-sm font-mono">
+                      <span>{option.contractName}</span>
+                      <span className="text-xs text-muted-foreground">{`.${option.moduleName}`}</span>
```

Comment:
Shouldn't the `contractName` be after the module? It should be read for example `synthetix.CoreProxy` instead of `CoreProxy.synthetix`
```suggestion
                      <span className="text-xs text-muted-foreground">{`${option.moduleName}.`}</span>
                      <span>{option.contractName}</span>
```

Reply:
changed